enterprise/cloud.oracle/src/org/netbeans/modules/cloud/oracle/actions/DeleteNodeAction.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.netbeans.modules.cloud.oracle.actions;
+
+import java.awt.event.ActionEvent;
+import java.io.IOException;
+import java.util.Collection;
+import javax.swing.AbstractAction;
+import javax.swing.Action;
+import org.openide.DialogDisplayer;
+import org.openide.NotifyDescriptor;
+import org.openide.awt.ActionID;
+import org.openide.awt.ActionReference;
+import org.openide.awt.ActionReferences;
+import org.openide.awt.ActionRegistration;
+import org.openide.nodes.Node;
+import org.openide.util.ContextAwareAction;
+import org.openide.util.Exceptions;
+import org.openide.util.Lookup;
+import org.openide.util.NbBundle;
+import org.openide.util.RequestProcessor;
+
+/**
+ *
+ * @author Dusan Petrovic
+ */
+@ActionID(
+        category = "Tools",
+        id = "org.netbeans.modules.cloud.oracle.actions.DeleteNodeAction"
+)
+@ActionRegistration(
+        displayName = "#DeleteNodeAction",
+        asynchronous = true,
+        lazy = true
+)
+
+@ActionReferences(value = {
+    @ActionReference(path = "Cloud/Oracle/ContainerTag/Actions", position = 250),
+    @ActionReference(path = "Cloud/Oracle/Vault/Secret/Actions", position = 250)
+})
+@NbBundle.Messages({
+    "DeleteNodeAction=Delete item",
+    "MSG_ConfirmDeleteAction=Are you sure that you want to delete {0}",
```

Comment:
Nobody funds (much) internationalization ;) these days  ... but still: please place a comment line above parametrized message. The recommended format is
```
# {n} - description of parameter n
```
It used to help translators to understand what individual parameters represent; they work primarily with .properties only.